ui-ngx/src/app/modules/home/components/alarm-rules/alarm-rule.module.ts

```diff
@@ -0,0 +1,94 @@
+} from "@home/components/alarm-rules/cf-alarm-rule-condition-dialog.component";
+import { CfAlarmScheduleComponent } from "@home/components/alarm-rules/cf-alarm-schedule.component";
+import { CfAlarmScheduleDialogComponent } from "@home/components/alarm-rules/cf-alarm-schedule-dialog.component";
+import {
+  EntityDebugSettingsButtonComponent
+} from "@home/components/entity/debug/entity-debug-settings-button.component";
+import { AlarmRuleFilterTextComponent } from "@home/components/alarm-rules/filter/alarm-rule-filter-text.component";
+import {
+  CalculatedFieldArgumentsTableModule
+} from "@home/components/calculated-fields/components/calculated-field-arguments/calculated-field-arguments-table.module";
+import {
+  AlarmRuleFilterPredicateListComponent
+} from "@home/components/alarm-rules/filter/alarm-rule-filter-predicate-list.component";
+import {
+  AlarmRuleFilterPredicateComponent
+} from "@home/components/alarm-rules/filter/alarm-rule-filter-predicate.component";
+import {
+  AlarmRuleFilterPredicateValueComponent
+} from "@home/components/alarm-rules/filter/alarm-rule-filter-predicate-value.component";
+import {
+  AlarmRuleComplexFilterPredicateDialogComponent
+} from "@home/components/alarm-rules/filter/alarm-rule-complex-filter-predicate-dialog.component";
+import { AlarmRuleFilterListComponent } from "@home/components/alarm-rules/filter/alarm-rule-filter-list.component";
+import { AlarmRuleFilterDialogComponent } from "@home/components/alarm-rules/filter/alarm-rule-filter-dialog.component";
+import { AlarmRuleDetailsDialogComponent } from "@home/components/alarm-rules/alarm-rule-details-dialog.component";
+
+@NgModule({
+  declarations: [
+    AlarmRuleDialogComponent,
+    CreateCfAlarmRulesComponent,
+    CfAlarmRuleComponent,
+    CfAlarmRuleConditionComponent,
+    CfAlarmRuleConditionDialogComponent,
+    CfAlarmScheduleComponent,
+    CfAlarmScheduleDialogComponent,
+    AlarmRuleFilterTextComponent,
+    AlarmRuleFilterListComponent,
+    AlarmRuleFilterDialogComponent,
+    AlarmRuleFilterPredicateListComponent,
+    AlarmRuleFilterPredicateComponent,
+    AlarmRuleFilterPredicateValueComponent,
+    AlarmRuleComplexFilterPredicateDialogComponent,
+    AlarmRuleDetailsDialogComponent,
+  ],
+  imports: [
+    CommonModule,
+    SharedModule,
+    EntityDebugSettingsButtonComponent,
+    CalculatedFieldArgumentsTableModule
+  ],
+  exports: [
+    AlarmRuleDialogComponent,
+    CreateCfAlarmRulesComponent,
+    CfAlarmRuleComponent,
+    CfAlarmRuleConditionComponent,
+    CfAlarmRuleConditionDialogComponent,
+    CfAlarmScheduleComponent,
+    CfAlarmScheduleDialogComponent,
+    AlarmRuleFilterTextComponent,
+    AlarmRuleFilterListComponent,
+    AlarmRuleFilterDialogComponent,
+    AlarmRuleFilterPredicateListComponent,
+    AlarmRuleFilterPredicateComponent,
+    AlarmRuleFilterPredicateValueComponent,
+    AlarmRuleComplexFilterPredicateDialogComponent,
+    AlarmRuleDetailsDialogComponent,
```

Comment:
```suggestion
    AlarmRuleDialogComponent,
```

Need to review. Must export only the used component